### PR TITLE
feat: Always enables googSuspendBelowMinBitrate for p2p.

### PR DESCRIPTION
**Pull request by bgrozev**

### CODE REVIEW

File: modules/RTC/RTCUtils.js [hunk 1]
```diff
@@ -872,6 +865,21 @@ class RTCUtils extends Listenable {
                         return this.audioTracks;
                     };
                 }
+
+                this.p2pPcConstraints
+                    = JSON.parse(JSON.stringify(this.pcConstraints));
+
+                // Allows sending of video to be suspended if the bandwidth
+                // estimation is too low.
+                if (!options.disableSuspendVideo) {
```
Comment (saghul) [inline modules/RTC/RTCUtils.js]: shouldn't this setting now go to the p2p section? or be removed altogether, if we'd like to have it on always?

Comment (bgrozev) [inline modules/RTC/RTCUtils.js]: No, the option affects the bridge connection. For p2p we enable it unconditionally.

Comment (saghul) [inline modules/RTC/RTCUtils.js]: Doh, ok, I missread, sorry!

**Comment by saghul** [comment]
Hopefully atlaskit is fixed soon so the CI works, I'll merge this then.

**Comment by bgrozev** [comment]
jenkins test this please

**Pull request merged**